### PR TITLE
NCBI specification

### PR DESCRIPTION
**Pull request by BusseChristian**
Converted the existing MiAIRR-to-NCBI docs to reStructuredText.
Included the modifications that came up during @javh's test submissions.
Completed split between submission manual and specification.

### CODE REVIEW

**Comment by javh** [comment]
I looked through the GenBank submission section.  It looks pretty solid to me.  A few minor things.

**1.** Two changes in the example GenBank record:

> KEYWORDS    TLS; Targeted Locus Study; AIRR; <other keywords>.
> DEFINITION  TLS: Mus musculus immunoglobulin heavy chain variable region, sequence.

I haven't actually done the TLS submission yet, so these are the only changes I know of right now.

**2.** I'm not sure whether it should be:

> /inference="COORDINATES:nucleotide motif:IgBLAST:1.6"

Or:

> /inference="alignment:IgBLAST:1.6"

Something to discuss.

**3.** Maybe we should mention that the qualifiers in 'source' of the GenBank file need to have identical values to those same qualifiers in the BioSample record. An example of the problem from my earlier test submission:

BioSample
```
tissue="Peripheral blood "
cell type="B cell"
isolate="HD07"
sex="male "
```
GenBank submission:

```
/isolate="HD07"
/cell_type="naive B cell"
```

Required them to make the following changes:

+ Changed the BioSample cell type to: "naive B cell" to match the Sequin submission
+ Added "tissue-type" and "sex" to the Sequin file to match the information in the BioSample.

**Comment by BusseChristian** [comment]
@javh: The latest commit includes all the changes we discussed today. Please have a look, thanks!

File: docs/miairr/manual_miairr_ncbi.rst [hunk 1]
```diff
@@ -122,8 +122,18 @@ files MUST ensure that the templates are identical to the current
...
 
 
-MiAIRR data submission to GenBank
----------------------------------
+MiAIRR data submission to GenBank/TLS
+-------------------------------------
+
+Processed sequence data will be submitted to the "Targeted Locus Study"
+(TLS) section of GenBank. The details of this submission process are
+currently still finalized. Basically the procedure is identical to a
```
Comment (javh) [inline docs/miairr/manual_miairr_ncbi.rst]: "still being finalized. Basically, the"

File: docs/miairr/manual_miairr_ncbi.rst [hunk 2]
```diff
@@ -122,8 +122,18 @@ files MUST ensure that the templates are identical to the current
...
+conventional GenBank submission with the exception of additional
+keywords marking it as TLS submission.
+
+Non-functional records should be removed before the data submission or
```
Comment (javh) [inline docs/miairr/manual_miairr_ncbi.rst]: "Non-productive"

File: docs/miairr/specification_miairr_ncbi.rst [hunk 3]
```diff
@@ -193,24 +194,40 @@ The header MUST contain all of the following elements:
...
+   -  the term "MiAIRR:<x>.<y>" with <x> and <y> indicating the used
+      version and subversion of the MiAIRR standard.
 
 -  REQUIRED: DOI of the associated free-text record containing the
-   information on data processing (MiAIRR data set 5) as ``RX`` line
-   (GenBank: ``REMARK`` within a ``REFERENCE`` [4]_.
+   information on data processing (MiAIRR data set 5) as ``REMARK``
+   within a ``REFERENCE`` [4]_ (ENA: ``RX`` line).
+
+-  OPTIONAL: The use of `structured records`_ is currently evalutated
```
Comment (javh) [inline docs/miairr/specification_miairr_ncbi.rst]: `structured comments` is currently being evaluated

File: docs/miairr/specification_miairr_ncbi.rst [hunk 4]
```diff
@@ -244,6 +261,27 @@ MUST or SHOULD contain the following keys/qualifiers:
    -  RECOMMENDED: qualifier ``/strain``, if ``/organism`` is "Mus
       musculus".
 
+Note that additional qualifiers might be REQUIRED by GenBank to
+harmonize the GenBank record with the BioSample referenced by it in the
+header. A list of known BioSample keyword and GenBank qualifiers that
+MUST contain the same information can be found below. Whether (and if
```
Comment (javh) [inline docs/miairr/specification_miairr_ncbi.rst]: "(and in which direction)"

**Comment by javh** [comment]
Looks good!  I made a few comments, but they are all just typographical. No content changes.

File: docs/miairr/manual_miairr_ncbi.rst (hunk 1 above)
Comment (BusseChristian) [reply docs/miairr/manual_miairr_ncbi.rst]: Fixed.

File: docs/miairr/manual_miairr_ncbi.rst (hunk 2 above)
Comment (BusseChristian) [reply docs/miairr/manual_miairr_ncbi.rst]: Fixed.

File: docs/miairr/specification_miairr_ncbi.rst (hunk 3 above)
Comment (BusseChristian) [reply docs/miairr/specification_miairr_ncbi.rst]: Fixed.

File: docs/miairr/specification_miairr_ncbi.rst (hunk 4 above)
Comment (BusseChristian) [reply docs/miairr/specification_miairr_ncbi.rst]: Fixed.

**Comment by BusseChristian** [comment]
Ok, all fixed.

**Pull request merged**